packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -971,6 +989,19 @@ describe("TableComponent", () => {
 			expect(component.getColumnSortingPriority("c")).toBeUndefined();
 		});
 	});
+	
+	describe("getRowIndex", () => {
+		it("should return the right index for every row", () => {
+			for (let i = 0; i < component.dataSource.data.length; i++) {
+				expect(component.getRowIndex(component.dataSource.data[i])).toBe(i+1);
+			}
+		});
+		
+		it("should return 'void' if dataSource is not initialized yet", () => {
+			component.dataSource = <any>undefined;
+			expect(component.getRowIndex({})).toBe();
```

Comment:
Small change: can you add some dummy data to the `{}` so it is clear that is just a row object?